.github/workflows/main.yml

```
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: [2.7, 3.7, 3.8, 3.9, "3.10", 3.11.0]
+        python-version: [2.7, 3.7, 3.8, 3.9, "3.10", 3.11.0, 3.12]
```

Use latest 3.11 (it's now up to 3.11.6) not pinned to 3.11.0:

```suggestion
        python-version: [2.7, 3.7, 3.8, 3.9, "3.10", "3.11", "3.12"]
```